src/EnergyPlus/WaterUse.cc

```diff
@@ -1052,8 +1052,8 @@ namespace WaterUse {
 
         if (this->TargetTempSchedule > 0) {
             this->TargetTemp = ScheduleManager::GetCurrentScheduleValue(state, this->TargetTempSchedule);
-        } else { // If no TargetTempSchedule, use all hot water
-            this->TargetTemp = this->HotTemp;
+        } else { // If no TargetTempSchedule, use all cold water
+            this->TargetTemp = this->ColdTemp;
```

Comment:
OK, so there is a TargetTempSchedule (mixed water temp), a ColdTempSchedule, and a HotTempSchedule. The Cold/HotTempSched's should have no impact on mixed water temp. So if no TargetTempSchedule exists then it does make sense to use the ColdTemp.

This logic suggests that there are times when the HotTemp should be used.

    waterEquipment.allowHotControl = (waterEquipment.TargetTempSchedule && waterEquipment.HotTempSchedule) || waterEquipment.Connections; 

Maybe this logic should include the allowHotControl variable?

        if (this->TargetTempSchedule > 0) {
            this->TargetTemp = ScheduleManager::GetCurrentScheduleValue(state, this->TargetTempSchedule);
        } else if (waterEquipment.allowHotControl )  { // If no TargetTempSchedule, use all hot water if applicable
            this->TargetTemp = this->HotTemp;
        } else { // If no TargetTempSchedule, use all cold water
            this->TargetTemp = this->ColdTemp;
        }

Reply:
Interesting notes in the idd:

    WaterUse:Equipment,
       \memo A generalized object for simulating all water end uses. Hot and cold water uses are
       \memo included, as well as controlled mixing of hot and cold water at the tap. The
       \memo WaterUse:Equipment object can be used stand-alone, or coupled into a plant loop using
       \memo the WaterUse:Connections object (see below). The WaterUse:Connections object allows
       \memo water uses to be linked to WaterUse:Storage objects to store and draw reclaimed water.
       \memo The object can also simulate drainwater heat recovery.
    A4 , \field Target Temperature Schedule Name
       \type object-list
       \object-list ScheduleNames
       \note Defaults to hot water supply temperature
    A5 , \field Hot Water Supply Temperature Schedule Name
       \type object-list
       \object-list ScheduleNames
       \note Defaults to cold water supply temperature
    A6 , \field Cold Water Supply Temperature Schedule Name
       \type object-list
       \object-list ScheduleNames
       \note Defaults to water temperatures calculated by Site:WaterMainsTemperature object

Reply:
So here it looks like that (first?) we will need to determine (according to idd, code) what the default target water temperature should go to for the "target temperature schedule = blank case": if determine that this should be default to cold water, then the IDD description needs to be changed here (as well as the IORef doc descriptions, which is already changed for the "Option 2" fix). In this case, if the 22.2 new feature/fix ( mixing cold/hot relationship) decided this should be the case, then we will change the IDD description as well.

If we determined that here that the target water temperature should take the hot water temperature, then we should go to the "Option 1" way of fix, to find ways to make the code to use hot water temperature as the default target temperature when the target temperature schedule field is blank. The good side is that it seems to be consistent to what the older versions (<=v22.1) did; and no IORef or IDD change is required.

Reply:
> 
Trying to figure out the proposed adding the logic of  
```
 } else if (waterEquipment.allowHotControl )  { // If no TargetTempSchedule, use all hot water if applicable
        this->TargetTemp = this->HotTemp;
```

1) Here the block is already the situation where target water temperature schedule is not present (0); then autoHotControl will be true only if there is water connection. So this logic basically will be true and executed only if there is water connection. If this is the case, and in this case the hot water temperature should be the default target temperature, then it is the right logic.

2) Then the next block of setting cold water temperature as the default target water temperature will only be executed when there is be no water connection (and no target water schedule temperature field, as it is also in the situation of target water temperature schedule is not present). 

So we will need to decide the logics above would be the correct  (or expected) logics.

Reply:
I do like this suggested logic:

    if (this->TargetTempSchedule > 0) {
        this->TargetTemp = ScheduleManager::GetCurrentScheduleValue(state, this->TargetTempSchedule);
    } else if (waterEquipment.allowHotControl )  { // If no TargetTempSchedule, use all hot water if applicable
        this->TargetTemp = this->HotTemp;
    } else { // If no TargetTempSchedule, use all cold water
        this->TargetTemp = this->ColdTemp;
    }

except that it relies on this logic being accurate:

    waterEquipment.allowHotControl = (waterEquipment.TargetTempSchedule && waterEquipment.HotTempSchedule) || waterEquipment.Connections;

Is it possible the user had other intentions with either of these inputs? Is there another corner case we are missing? If there is any doubt about the above logic then the user would have to provide an input to specify the control. Maybe the suggestion at the top of this comment is sufficient? and we wait for the next corner case to pop up. Does this suggestion fix the defect file?